lib/agent/agent-glue.c

```diff
@@ -218,6 +220,51 @@ frida_ansi_string_to_utf8 (const gchar * str_ansi, gint length)
 
 #ifdef HAVE_DARWIN
 
+void
+_frida_agent_thread_suspend_monitor_remove_cloaked_threads (FridaAgentThreadSuspendMonitor * self, task_inspect_t task, thread_act_array_t * threads, mach_msg_type_number_t * count)
```

Comment:
If we can make it static:

```suggestion
_frida_agent_thread_suspend_monitor_remove_cloaked_threads (task_inspect_t task, thread_act_array_t * threads, mach_msg_type_number_t * count)
```